README.md

```diff
@@ -267,15 +267,16 @@ our code should correctly handle it and return an error, e.g.
 
 ## Sanitizer support
 
-If your code uses [`fill_uninit`] and you enable memory sanitization
-(i.e. `-Zsanitizer=memory`), you need to pass the `getrandom_sanitize`
-configuration flag to enable unpoisoning of the destination buffer
-filled by `fill_uninit`.
+If your code uses [`fill_uninit`] and you enable
+[MemorySanitizer](https://doc.rust-lang.org/beta/unstable-book/compiler-flags/sanitizer.html#memorysanitizer)
+(i.e. `-Zsanitizer=memory`), we will automatically handle unpoisoning
+of the destination buffer filled by `fill_uninit`.
 
-For example, it can be done as follows (requires a Nightly compiler):
+The following command runs all our tests with MemorySanitizer
+(requires a Nightly compiler):
 ```sh
-RUSTFLAGS="-Zsanitizer=memory --cfg getrandom_sanitize" \
-    cargo test -Zbuild-std --target=x86_64-unknown-linux-gnu
+RUSTFLAGS="-Zsanitizer=memory" cargo test \
+    -Zbuild-std --target=x86_64-unknown-linux-gnu --all-targets
```

Comment:
This line break can be probably removed.

Reply:
I initially had this because the doc tests weren't working, but I realized this can be fixed by just setting `RUSTDOCFLAGS`. I've updated the docs and the CI invocation.

With the increased environment variables, I decided to keep the line-break as otherwise the wrapping ends up being hard to read.

Reply:
This line is intended for users, no? They probably do not need the `RUSTDOCFLAGS` flag.

Reply:
It may be worth to write this part like this:
> You can run sanitizer tests for your crate dependent on `getrandom` like this:
> ```sh
> RUSTFLAGS="-Zsanitizer=memory" cargo test -Zbuild-std --target=x86_64-unknown-linux-gnu
> ```

Reply:
> It may be worth to write this part like this:
> 
> > You can run sanitizer tests for your crate dependent on `getrandom` like this:
> > ```shell
> > RUSTFLAGS="-Zsanitizer=memory" cargo test -Zbuild-std --target=x86_64-unknown-linux-gnu
> > ```

That invocation fails without also setting `RUSTDOCFLAGS="-Zsanitizer=memory"`, which is why I updated the CI to set that environment variable as well.

Reply:
Oh I'm being dumb, I understand what you mean. Our crate needs `RUSTDOCFLAGS`, but other crates may not.

Reply:
Fixed the README to use your suggestion.

Reply:
Hm, it's a bit weird that we have to pass `RUSTDOCFLAGS` sedately. In that case, I am fine with both variants.